### DISCUSSION TITLE
Is package rules for Gradle not working?

### DISCUSSION THREAD

**Discussion by Kantis** [renovatebot/renovate#15743] [Request Help] [answered]
Hi! We recently started using Renovate for the Kotest project. We're trying to get rid of a bump of Junit that Renovate keeps opening (immortal) https://github.com/kotest/kotest/pull/3016

I tried adding a [package rule](https://github.com/kotest/kotest/blob/master/renovate.json) that I thought would mandate dependency dashboard approval before Renovate would open another PR for JUnit. 
```json
      {
         "matchPackagePrefixes": [ "org.junit"],
         "dependencyDashboardApproval": true
      }
```

Am I doing something wrong? 😄

**Comment by viceice** [comment, marked as answer]
you need to close the PR and delete the branch

**Reply by rarkins** [reply]
Make sure any existing PR is closed, and the branch deleted at the same time.

**Reply by Kantis** [reply]
Thanks! Previous PR must've been closed, but with branch remaining, or something.. 😄
